{
    "logs": [
        {
            "no": 1,
            "output": "ONLINE   101 online!",
            "time": "10-04-2025 13:52",
            "user": "101"
        },
        {
            "no": 2,
            "output": "ONLINE   101 online!",
            "time": "10-04-2025 13:52",
            "user": "101"
        },
        {
            "no": 3,
            "output": "WARN   Main thread error occured:\tHTTPConnectionPool(host='192.168.29.154', port=5000): Max retries exceeded with url: /command (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001FE2B0F6F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))",
            "time": "10-04-2025 13:53",
            "user": "101"
        },
        {
            "no": 4,
            "output": "ONLINE   03 online!",
            "time": "10-04-2025 13:54",
            "user": "03"
        },
        {
            "no": 5,
            "output": "WARN   Main thread error occured:\tHTTPConnectionPool(host='192.168.29.154', port=5000): Max retries exceeded with url: /command (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000019234D06F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))",
            "time": "10-04-2025 13:55",
            "user": "03"
        },
        {
            "no": 6,
            "output": "WARN   Main thread error occured:\tHTTPConnectionPool(host='192.168.29.154', port=5000): Max retries exceeded with url: /command (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000019234D06580>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))",
            "time": "10-04-2025 13:55",
            "user": "03"
        }
    ]
}